harness/determined/cli/trial.py

```diff
@@ -237,6 +243,35 @@ def trial_logs(args: Namespace) -> None:
         )
 
 
+def set_log_retention(args: Namespace) -> None:
+    if not args.forever and not isinstance(args.days, int):
+        raise cli.errors.CliError(
+            "Please provide an argument to set log retention. --days sets the number of days to"
+            + " retain logs from the time of creation, eg. `det t set log-retention 1 "
```

Comment:
Last comment, I promise. You don't actually need the `+` in these string concatenations. You can see an example of this elsewhere in this file.

Reply:
done!